botogram/defaults.py

```diff
@@ -95,8 +95,13 @@ def _help_generic_message(self, bot, commands):
                 summary = escape_html(command.summary)
                 if summary is None:
                     summary = "<i>%s</i>" % bot._("No description available.")
-                message.append("/%s <code>-</code> %s" %
-                               (command.name, summary))
+                parameters_list = command.parameters_list
```

Comment:
why you create new variable instead of using the same

Reply:
Done in c4f983b. Good ol' days